src/5e-SRD-Monsters.json

```diff
@@ -12602,7 +13609,91 @@
     "actions": [
       {
         "name": "Multiattack",
-        "desc": "The erinyes makes three attacks"
+        "desc": "The erinyes makes three attacks",
+        "options": {
```

Comment:
Nice combinatorics! :D

Reply:
Actually, another way we could do this is:
```
{
  "choose": 3,
  "from": [
    [
      {
        "name": "Longsword",
        "count": 1,
        "type": "melee"
      }
    ],
    [
      {
        "name": "Longbow",
        "count": 1,
        "type": "ranged"
      }
    ],
    [
      {
        "name": "Variant: Rope of Entanglement",
        "count": 1,
        "type": "ability"
      }
    ]
  ]
}
```

Reply:
That utilizes the choose format a bit better and makes it much cleaner.

Reply:
OMG, that seems so obvious now, well that is a far better syntax I will improve that

Reply:
Is being able to choose more than one of the same option valid for multi-attacks across all monsters? The idea I was going with originally was that you always choose one of the available options until you've hit the "choose: x" limit.

Reply:
Not for all but it is for some. There are a few where it says it has to be unique. I'm not sure the best way to handle that case though.

Reply:
A flag on the options object, something like "choose_unique:" true/false?